curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-hobbies-and-interests/657e76d866ebcec0e44d0748.md

```diff
@@ -3,21 +3,17 @@ id: 657e76d866ebcec0e44d0748
 title: Task 29
 challengeType: 19
 dashedName: task-29
-audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
 ---
 
-<!--
-AUDIO REFERENCE: 
-Jake: I haven't been on a bike since I was a kid.
--->
+<!-- (Audio) Jake: I haven't been on a bike since I was a kid. -->
 
 # --description--
 
-In the Present Perfect Continuous Tense, to make a sentence negative, you add `not` after `have` or `has`. This way, you can talk about something that has not been happening. 
+In the Present Perfect Tense, to make a sentence negative, you add `not` after `have` or `has`. This way, you can talk about something that has not occurred lately. 
 
-For example, if Jake wants to say he hasn't been cycling since he was a child, he can say, `I have not been cycling since I was a child.` This means he did not cycle from his childhood to now.
+For example, if Jake wants to say it's been a long time since he was on a bike for the last time, he can say, `I have not been on a bike since I was a child.`.
```

Comment:
```suggestion
For example, if Jake wishes to express that a significant amount of time has passed since his last bicycle ride, he could say, `I have not been on a bike since I was a child.`
```